postcss.config.js

```diff
@@ -0,0 +1,7 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+module.exports = {
+  plugins: [require('autoprefixer')({ grid: 'autoplace' }), require('postcss-discard-comments')],
```

Comment:
Getting this on build:
![image](https://user-images.githubusercontent.com/81580355/132193617-de4507ca-4e55-4696-b8ee-7cccba275187.png)